ocs_ci/deployment/cnv.py

```
@@ -385,6 +385,45 @@ def post_install_verification(self, raise_exception=False):
+                func=lambda: ocp.get(resource_name=constants.KUBEVIRT_HYPERCONVERGED)
+                .get("status", {})
+                .get("systemHealthStatus"),
+            ):
+                logger.info(f"HyperConverged systemHealthStatus: {sample}")
+                if sample == "healthy":
+                    logger.info("HyperConverged cluster is healthy")
+                    return
+        except exceptions.TimeoutExpiredError:
+            raise exceptions.TimeoutExpiredError(
+                f"HyperConverged did not reach 'healthy' status within {timeout}s"
+            )
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)